tests/test_circuit_validation.py

```diff
@@ -288,26 +290,146 @@ def test_no_edges_h5():
         assert errors == [Error(Error.FATAL, 'No "edges" in {}.'.format(edges_file))]
 
 
-def test_no_required_edge_population_datasets():
+def test_no_required_edge_population_datasets_one_group():
+    required_datasets = sorted([
+        'edge_type_id', 'source_node_id', 'target_node_id'])
+    with _copy_circuit() as (circuit_copy_path, config_copy_path):
```

Comment:
should we also have here `optional_datasets = ['edge_group_id', 'edge_group_index']`? We will also delete them
```
            for ds in optional_datasets:
                del h5f['edges/default/' + ds]
```
And they should not appear as errors. Or should they?

Reply:
They should not appears as error. I think a test is missing.